test/integration/proxy_test.js

```diff
@@ -715,7 +715,11 @@ test('reroutes retry to remote', function t(assert) {
             retrySchedule: [1]
         }, function onRequest(err, res) {
             assert.ifError(err, 'no error occurs');
-            assert.equal(res.body, 'rerouted', 'response from rerouted request is correct');
+            assert.equal(
+                res.body.toString('hex'),
+                new Buffer('rerouted').toString('hex'),
+                'response from rerouted request is correct'
```

Comment:
This test still breaks because its a unit test using an hammock. Basically `hammock` is used both in ringpop proxying and in the tests themself.

Since the `res` instance is allocated inside the test cluster we need to change the test.